OOP/InputAction.java

```diff
@@ -0,0 +1,105 @@
+import java.util.Scanner;
+
+public class InputAction extends Menu {
+	private static Train.Type type;
+
+	public InputAction() {
+
+		addMenuItem(new ActionableMenuItem("1-passenger", new PassengerAction()));
```

Comment:
You should not give serial nos. to the options means instead of "1-passenger" it should be "passenger". The serial nos. should be added in the "Menu" class's display method, as you are iterating the Menuitems list and displaying them one by one. That's a very simple job :)